package.json

```diff
@@ -69,7 +70,9 @@
   },
   "jest": {
     "moduleNameMapper": {
-      "^utils": "<rootDir>/src/utils"
+      "^utils": "<rootDir>/src/utils",
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/scripts/test/fileMock.js",
+      "\\.(css|less)$": "<rootDir>/scripts/test/styleMock.js"
```

Comment:
What's this one for out of curiousity? I think we're still css-free for snacks so far

Reply:
I added this for the svgs mostly. This is the generic style mocking that jest suggests